packages/transaction_signing_gateway/pubspec.yaml

```
@@ -17,6 +17,7 @@ dependencies:
   encrypt: ^5.0.1
   alan: ^0.40.2+1
   uuid: ^3.0.5
+  decimal: 1.5.0
```

since transaction_signing_gateway is a public package, let's not be too strict about versioning of the libraries it uses, otherwise its going to cause a lot of headache for dependency resolving in the client's code.
```suggestion
  decimal: ^1.5.0
```

---

Ahh noted

---

Added fix